app/controllers/evaluations_controller.rb

```diff
@@ -22,12 +26,148 @@ def submissions
     @submissions_count = helpers.calculate_submissions_count(@assigned_submissions)
   end
 
+  def show; end
+
+  def new
+    @evaluator_submission_assignment = find_evaluator_submission_assignment
+
+    if @evaluator_submission_assignment.nil? || !can_access_evaluation?
+      return redirect_to evaluations_path, alert: I18n.t("evaluations.alerts.evaluator_submission_assignment_not_found")
+    end
+
+    @evaluation_form = EvaluationForm.find_by(phase: @evaluator_submission_assignment.phase)
+
+    if @evaluation_form.nil?
+      return redirect_to evaluations_path, alert: I18n.t("evaluations.alerts.evaluation_form_not_found")
+    end
+
+    build_evaluation
+
+    render :new
+  end
+
+  def edit
+    @evaluation = Evaluation.find_by(id: params[:id])
```

Comment:
this needs to load the evaluator_submission_assignment otherwise it always returns unauthorized (the specs for this request route aren't checking the response status)